cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -45,17 +51,29 @@ public StepExecution start(StepContext context) throws Exception {
         return new ParallelStepExecution(this, context);
     }
 
+    /*package*/ boolean isFailFast() {
+        return failFast;
+    }
+
+
     @PersistIn(PROGRAM)
     static class ResultHandler implements Serializable {
         private final StepContext context;
+        private final ParallelStepExecution stepExecution;
```

Comment:
Unclear to me if storing a reference to the `ParallelStepExecution` like this is kosher. @kohsuke? I was already confused by why `ParallelStepExecution` is storing its `BodyExecution`s.